app/javascript/hooks/useTranslate.tsx

```diff
@@ -49,7 +50,8 @@ const useTranslate = () => {
 
   useEffect(() => {
     if (languageInRoute && googleTranslateDropdownElHasBeenRendered) {
-      const selectInDom = document.querySelector("select.goog-te-combo")
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const selectInDom: any = document.querySelector("select.goog-te-combo")
```

Comment:
See previous comment about `any` types.

Reply:
Created a ticket to track this effort: https://sfgovdt.jira.com/browse/DAH-1581